web/auth.html

```diff
@@ -1,21 +1,24 @@
 <!DOCTYPE html>
 <title>Authentication complete</title>
-<p>Authentication is complete. If this does not happen automatically, please close the window.</p>
+<p>Authentication is complete. If this page does not navigate automatically, please <a href="#" id="manualRedirectLink">click here</a> to continue.</p>
 <script>
   function postAuthenticationMessage() {
-    const message = {
-      'flutter-web-auth-2': window.location.href
-    };
-
-    if (window.opener) {
-      window.opener.postMessage(message, window.location.origin);
-      window.close();
-    } else if (window.parent && window.parent !== window) {
-      window.parent.postMessage(message, window.location.origin);
-    } else {
-      localStorage.setItem('flutter-web-auth-2', window.location.href);
-      window.close();
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
try catch

Reply:
Also add function description to make sure the input/output
Ex:
![image](https://github.com/linagora/twake-on-matrix/assets/80142234/4a730a0b-6f05-44e0-af3b-1ff1197c7a22)